picky-signtool/Cargo.lock

```diff
@@ -0,0 +1,1494 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Delete this Cargo.lock and add `picky-signtool` in workspace.

Reply:
`picky-signtool` is already in the workspace

Reply:
Indeed, just delete `picky-signtool/Cargo.lock` and make sure it isn’t generated again for whatever reason it was generated here